deluge/ui/gtkui/mainwindow.py

```diff
@@ -294,7 +295,7 @@ def on_vpaned_position_event(self, obj, param):
         self.config["window_pane_position"] = self.vpaned.get_position()
 
     def on_drag_data_received_event(self, widget, drag_context, x, y, selection_data, info, timestamp):
-        log.debug("Selection(s) dropped on main window %s", selection_data.data)
+        log.debug("Selection(s) dropped on main window %s", selection_data)  # fixed AttributeError: 'SelectionData' object has no attribute 'data'
```

Comment:
I don't think that is the correct fix, it think it's likely to be: `selection_data.get_text()`